http/middleware.go

```diff
@@ -42,3 +42,17 @@ func Logger(next http.Handler) http.Handler {
 		next.ServeHTTP(w, r)
 	})
 }
+
+func CORS(next http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Access-Control-Allow-Origin", "*")
```

Comment:
Do we really want a wildcard here?
Apparently this restricts any credential support including cookies.
Wouldn't this make our authorization unusable?